lib/json_api_resource/resource.rb

```diff
@@ -1,3 +1,12 @@
+require 'active_support'
+require 'active_support/concern'
+require 'active_support/core_ext/module'
+require 'active_support/core_ext/class/attribute'
+require 'active_model'
+require 'active_model/model'
+require 'active_model/validations'
+require 'active_model/callbacks'
```

Comment:
Why do we need this? I'm not sure if this is already tied to rails, but do we need to tie this directly to rails?